app-containers/crun/crun-1.9.1.ebuild

```
@@ -0,0 +1,73 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
```

```suggestion
PYTHON_COMPAT=( python3_{10..12} )
```
IF 3.12 works. 3.9 is disabled already in the repo.

---

does the fixup commit work for you?